channels/1ds-post-js/src/HttpManager.ts

```diff
@@ -675,6 +675,10 @@ export class HttpManager {
                 _headers[name] = value;
             };
 
+            _self.removeHeader = (name: string) => {
+                delete _headers[name];
```

Comment:
Is it possible _headers is null? this could throw an exception I believe

Reply:
it shouldn't ever be :-) as it's initialized in the `_initDefaults` to an empty object.